src/app/business/home/home.component.ts

```diff
@@ -82,15 +83,26 @@ export class HomeComponent implements OnInit {
         this.getCounts();
         this.getType();
         
+  if(this.isLocalCloud==true){this.backendForm = this.fb.group({
```

Comment:
The type of "this.isLocalCloud" is Boolean. There is no need to write "this.isLocalCloud" == true